bazel/README.md

```diff
@@ -160,11 +160,12 @@ bazel test //test/common/http:async_client_impl_test --strategy=TestRunner=stand
 # Stack trace symbol resolution
 
 Envoy can produce backtraces on demand and from assertions and other fatal
-actions like segfaults. The stack traces written in the log or to stderr contain
-addresses rather than resolved symbols. The `tools/stack_decode.py` script exists
-to process the output and do symbol resolution to make the stack traces useful. Any
-log lines not relevant to the backtrace capability are passed through the script unchanged
-(it acts like a filter).
+actions like segfaults. Where supported, stack traces will contain resolved
+symbols. On systems where absl::Symbolization is not supported, the stack
+traces written in the log or to stderr contain addresses rather than resolved symbols.
+The `tools/stack_decode.py` script exists to process the output and do symbol resolution
```

Comment:
Do we still support `stack_decode.py` where `absl::Symbolization` is supported? I think the former gives you line numbers..

Reply:
The former didn't work at all for me (despite copy-pasting all the various tags from the instructions) which is why I finally got around to doing it inline.

Are you requesting a mode where we disable it?  I could add a flag.

Reply:
I'm more curious if you can use both together.. Is the `absl::` approach able to extract line numbers when we compile `-c dbg`? That would totally obviate `stack_decode.py`.

Reply:
Unfortunately the traces I posted were with -c dbg so I'm going to go with "no"

I've already tracked down the guy who made these changes in absl, so I can go ask them if they're planning on enhancements.  Meanwhile I've made it so stack decode works as well as it has, which is to say not at all for me, but I'm fairly confident it will no longer regress for those who get useful output from it.  Before my PR I'd get something of the form
https://pastebin.com/x1KWJjFg
and with my PR and without the latest push it was like
https://pastebin.com/23KrqEmT
witht he latest push it's back to the former   #1 ?? ??:0 output which is as much as I can verify :-/

Reply:
Yeah, `./tools/stack_decode.py` seems to be broken, and I can't get anything out of it.

@alyssawilk Your "good" output is actually output from `absl::Symbolize()` that wasn't matched against any regex and left as-is. The `./tools/stack_decode.py` output starts with `Backtrace (most recent call first) from thread 0:`